src/Accounts/Accounts/help/Disable-AzContextAutosave.md

```diff
@@ -22,13 +22,20 @@ Turn off autosaving Azure credentials.  Your login information will be forgotten
 
 ## EXAMPLES
 
-### Disable autosaving the context
-```
+### Example 1: Disable autosaving the context
+```powershell
 PS C:\> Disable-AzContextAutosave
 ```
 
 Disable autosave for the current user.
 
+### Example 2
+```powershell <!-- Aladdin Generated Example --> 
+PS C:> Disable-AzContextAutosave -Scope Process
+```
+
+Turn off autosaving Azure credentials. (autogenerated)
```

Comment:
`Turn off autosaving Azure credentials in current powershell session. (autogenerated)` maybe better.